package.json

```diff
@@ -24,7 +24,7 @@
   "dependencies": {
     "kew": "0.7.0",
     "typ": "0.6.3",
-    "aws-sdk": "2.2.39"
+    "aws-sdk": "2.2.47"
```

Comment:
FWIW, I believe that you can use the `^` notation for minor versions to be included automatically.

cc @kylehg